catalog/core/catalog-core-camelcomponent/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,32 +15,34 @@
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
            xsi:schemaLocation="
   http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd"
-        >
+>
 
     <reference id="transformerMapper" interface="ddf.mime.MimeTypeToTransformerMapper"/>
 
     <reference id="mimeTypeMapper" interface="ddf.mime.MimeTypeMapper"/>
 
-    <reference id="framework" interface="ddf.catalog.CatalogFramework"/>
+    <reference id="framework" interface="ddf.catalog.CatalogFramework" availability="optional"/>
```

Comment:
My concern is if the code that is using the `framework` reference is protecting for when it is not available. If the `catalogComponent` bean were to be called before the framework is available will it handle that gracefully. Same comment for the other references that were changed to `optional`.